internal/pkg/agent/cmd/diagnostics.go

```diff
@@ -156,6 +163,72 @@ func createZip(fileName string, agentDiag []client.DiagnosticFileResult, unitDia
 	return closeHandlers(nil, zw, f)
 }
 
+func writeRedacted(streams *cli.IOStreams, fullFilePath string, fr client.DiagnosticFileResult, w io.Writer) error {
+	out := &fr.Content
+
+	// Should we support json too?
+	if fr.ContentType == "application/yaml" {
+		unmarshalled := map[string]interface{}{}
+		err := yaml.Unmarshal(fr.Content, &unmarshalled)
+
+		if err != nil {
+			// Best effort, output a warning but still include the file
+			fmt.Fprintf(streams.Err, "[warning] Could not redact %s due to unmarshalling error: %s\n", fullFilePath, err)
```

Comment:
[Suggestion]
Make it upper case in the hope it'll help it'll draw better people's attention

```suggestion
			fmt.Fprintf(streams.Err, "[WARNING] Could not redact %s due to unmarshalling error: %s\n", fullFilePath, err)
```